homestar-runtime/src/worker.rs

```diff
@@ -209,8 +219,26 @@ where
                     )?;
                 }
 
+                info!(
+                    subject = "worker.start_workflow",
+                    category = "worker.run",
+                    workflow_cid,
+                    "starting workflow"
+                );
+
                 // Run the queue of tasks.
-                self.run_queue(ctx.scheduler, running_tasks).await
+                let result = self.run_queue(ctx.scheduler, running_tasks).await;
+
+                if let Ok(()) = result {
+                    info!(
```

Comment:
-> move this to where we complete the status at the end of batches.

Have the log captured by the span for replaying receipts, which will happen whether or not the workflow is complete or not.

Reply:
Done 👌